EsoTech.MessageQueue/AzureServiceBus/AzureServiceBusNamingConvention.cs

```diff
@@ -34,5 +42,23 @@ private static string GetServiceName(Type type)
 
             return typeFullName.Split('.').Skip(1).First().ToLower();
         }
+
+
+        private string GetTypeNamePrefix(Type mt, int maxLength)
+        {
+            string prefix = Truncate(mt.Name, maxLength);
```

Comment:
You could use Type.ToString(), it returns nicely formatted type name for generics.